src/autoapp/Projection/QtAudioOutput.cpp

```diff
@@ -55,12 +60,15 @@ void QtAudioOutput::createAudioOutput()
 
 bool QtAudioOutput::open()
```

Comment:
Rather than patching this to always return `true`, I think it's perhaps a better idea to remove this function and don't call it.

Reply:
The base class has a pure virtual function here, so it cannot be removed in `QtAudioOutput`.  `RtAudioOutput` does more in this function and might return `true` or `false`.
https://github.com/opencardev/openauto/blob/fd5ce7ac076a56eced9fc78c895ef1f21aa641f3/include/f1x/openauto/autoapp/Projection/IAudioOutput.hpp#L42

Reply:
Thanks for the clarification!